docs/source/Advanced-Usage.md

```diff
@@ -196,7 +196,7 @@ Some analyzers could require a special configuration:
   - `/integrations/pcap_analyzers/config/suricata/rules`: here there are Suricata rules. You can change the `custom.rules` files to add your own rules at any time. Once you made this change, you need to either restart IntelOwl or (this is faster) run a new analysis with the Suricata analyzer and set the parameter `reload_rules` to `true`.
   - `/integrations/pcap_analyzers/config/suricata/etc`: here there are Suricata configuration files. Change it based on your wish. Restart IntelOwl to see the changes applied.
 
-- `Yara_Scan_Custom_Signatures`: you can use this pre-defined analyzer to run your own YARA signatures, either custom or imported. Just upload the `.yar` files with the signatures in the directory `/configuration/custom_yara`. That directory is mounted as a bind volume in Docker so you do not need to do anything to see the changes in the application.
+- `Yara`: you can customize the `repositories` parameter to download and use different rules from the default that IntelOwl currently support. The rules *must* be downloaded: this means that local rules are not supported, allowing to sync the rules in every celery worker without the usage of shared volumes.  
```

Comment:
I understand that all of this is required for cluster deployments and it is a smart way to handle the problem.

However we are strill dropping the support for the majority of use cases which is researchers with single deployments which do not want to install yara separately and just want to leverage the one already installed in IntelOwl. IntelOwl could comes handy when doing testing of Yara signatures in that sense.

I would still keep support for the old `/configuration/custom_yara` directory for single deployments. We could explicitly say here that it is not intended to be used in cluster deployments so the users knows how to use it.

Reply:
I do not think that we are "dropping the support": upload your rules on github is a fair enough condition, even if you are actively developing them, since you ca have as URL link in the secret the `develop` branch and maintain them synced more easily than before